test/e2e/tests/tests.go

```diff
@@ -14,4 +14,6 @@ var (
 	ConformanceTests   []suite.ConformanceTest
 	UpgradeTests       []suite.ConformanceTest
 	MergeGatewaysTests []suite.ConformanceTest
+	InternetGCTests    []suite.ConformanceTest
```

Comment:
not a fan of this naming. should we just go as `MultipleGCTests` with type `map[string][]suite.ConformanceTest`, the GC name can be used as the map key ?

Reply:
I agree, let's change the name for the tests, but I will keep this scenario naming just to keep up with the gw-api example